cloud_launch_small.json

```diff
@@ -18,6 +18,23 @@
           "cols": 1,
           "rows": 1
         }
+      },
+      {
```

Comment:
since we've added the `SimulatedPlayerCoordinator` to `cloud_launch_small` - we could probably remove the `cloud_launch_small_simulated_players` config